lib/datadog/tracing/correlation.rb

```diff
@@ -48,7 +48,7 @@ def initialize(
           # Dup and freeze strings so they aren't modified by reference.
```

Comment:
Let's document that "this class is for usage with log correlation. To continue from a trace, users should use TraceDigest instead."